app/scripts/views/components/barChart/barChart.js

```diff
@@ -131,8 +154,14 @@
           _returnValue = 'NA';
         }
       } else if (v === opts_.xDomain[0]) {
+        if (data_.smallDataFlag) {
+          return '<=' + e(opts_.xDomain[1]);
+        }
         return '<=' + opts_.xDomain[1];
       } else if (v === opts_.xDomain[opts_.xDomain.length - 2]) {
+        if (data_.smallDataFlag) {
```

Comment:
var e = d3.format('.1e');// convert small data to scientific notation format
For scientific numbers, they should be covert to scientific format by e(). If I merged them, normal numbers will also show in scientific format.